test/format/config_def.c

```diff
@@ -129,6 +129,12 @@ CONFIG configuration_list[] = {
   {"disk.mmap_all", "configure mmap operations (read and write)",
     C_BOOL, 5, 0, 0, V_GLOBAL_DISK_MMAP_ALL},
 
+  {"file_manager.close_handle_minimum", "number of handles open before the file manager will look for handles to close",
+    0x0, 0, 50, 100, V_GLOBAL_FILE_MANAGER_CLOSE_HANDLE_MINIMUM},
```

Comment:
Can the minimum allowed to be 0? Should it be at least we have 1 handle open to check for handles to close?